cnv_release_notes/cnv_release_notes.adoc

```diff
@@ -16,11 +16,21 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
 [[RN_known_issues]]
 == Known issues
 
-* When using `virtctl vnc $vmi` to show the VM’s display, the Remote
-Viewer may indefinitely show the message "Waiting for display 1…"
-https://bugzilla.redhat.com/show_bug.cgi?id=1619218[BZ1619218]
-
 * After creating a PVC with CDI annotation, if you attach the PVC to 
 a VM object and start the VM before allowing CDI to complete, the VM 
 may erroneously be listed as `Running` with no errors shown in the web 
-console. https://bugzilla.redhat.com/show_bug.cgi?id=1640505[BZ1640505]
+console. https://bugzilla.redhat.com/show_bug.cgi?id=1640505[(BZ#1640505)]
+
+* In the Create VM Wizard, using the `PXE` source option with the `Start VM on 
+creation` option may result in the boot order not changing after stopping 
+and starting the VM. The VM may still boot from the PXE NIC instead of the 
```

Comment:
'may still' is not accurate, 
it will always boot from PXE, until you stop and start the vm